airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -273,7 +273,7 @@ const useInitialValues = (
       syncCatalog: initialSchema,
       schedule: connection.schedule !== undefined ? connection.schedule : DEFAULT_SCHEDULE,
       prefix: connection.prefix || "",
-      namespaceDefinition: connection.namespaceDefinition,
```

Comment:
I'm not familiar with this area of the code, but should it try to use the `namespaceDefinition` of the connection first before falling back to the default? Have you tested this when both creating and also editing a new connection?

Reply:
Yes it should be!